### PR TITLE
Fix a JPEG vips issue

### PR DESCRIPTION
**Pull request by manthey**
JFIF headers are now conditional.

See if an iOS is requesting a tile.  If so and we don't have a JFIF header, convert the JPEG to a PIL image and then to a new JPEG rather than adding a JFIF header.

It turns out that the colorspace of a JPEG is complicated, and adding a JFIF header changes it in unexpected ways.  See the comments.  vips (especially latest stable) often generates RGB jpegs (not YCbCr).  If you add a JFIF header to this, the colorspace is implied to be YCbCr, which is wrong.  Therefore, if we want to add a JFIF header, we *must* convert the colorspace, too.

Increase output from the create_tif script to make it easier to debug.   This print the actual command used and always prints non-empty stdout and stderr.

### CODE REVIEW

**Comment by manthey** [comment]
@zachmullen Could you review this PR when you get a chance?

File: server/create_tiff.py [hunk 1]
```diff
@@ -30,6 +30,8 @@
 
 convert_command = (
     'vips',
+    # '--vips-concurrency', '1',
+    # '--vips-progress',
```
Comment (zachmullen) [inline server/create_tiff.py]: Did you mean to leave these commented lines here?

File: plugin_tests/common.py [hunk 2]
```diff
@@ -29,6 +29,7 @@
 from girder.constants import SortDir
 
 
+JFIFHeader = '\xff\xd8\xff\xe0\x00\x10JFIF'
```
Comment (zachmullen) [inline plugin_tests/common.py]: Should this be `b\xff\xd8\xff\xe0\x00\x10JFIF`? (Same for the others below?) That would prevent certain undesired behavior between python2 and 3 when doing comparisons.

File: server/create_tiff.py (hunk 1 above)
Comment (manthey) [inline server/create_tiff.py]: I did, as optionally enabling them has helped with tracking down vips issues on certain installations.  I should add a comment to that effect.

File: plugin_tests/common.py (hunk 2 above)
Comment (manthey) [inline plugin_tests/common.py]: Yes, it should.  Thanks.

**Pull request merged**